cmd/aries-js-worker/src/worker-impl-rest.js

```diff
@@ -67,4 +74,18 @@ function newResponse(id, payload, errMsg, topic) {
         errMsg: errMsg,
         topic: topic
     };
-}
\ No newline at end of file
+}
+
+const wsnotifier = class {
+    constructor(url, postMsg) {
+        this.socket = new WebSocket(url);
+        this.socket.addEventListener('message', function (event) {
+            // TODO REST agents are not currently revealing topic information on incoming messages,
```

Comment:
can you patch the REST agent?

Reply:
as part of https://github.com/hyperledger/aries-framework-go/issues/1323

All BDD tests will fail, so will push it in next PR.

Reply:
ok